web/style.css

```diff
@@ -281,6 +282,7 @@ input[type="range"]::-ms-fill-upper {
   background-color: var(--text);
   z-index: -1;
   transform: translate(2px, 2px);
+  width: 3rem;
```

Comment:
would u mind elaborating why this fixed width is needed? it's not immediately obvious to me, so i must have missed sth.

Reply:
sorry, it's just that when the text changes from 4 chars (play) to 5 (pause), the button like "jumps" as the width is different, which imo defeats the physical button effect.

i don't really write css and just eyeballed it, but the fix is to explicitly define a width for the button